### PR TITLE
jigsaw: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/jigsaw.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# jigsaw
+
+> A Laravel-based static site builder for PHP.
+> More information: <https://jigsaw.tighten.co>.
+
+- Initialise a project:
+
+`jigsaw init`
+
+- Initialise a project with a specific preset:
```
Comment (andrik) [inline pages/common/jigsaw.md]:
```suggestion
- Initialise a project using a starter template:
```

File: pages/common/jigsaw.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`jigsaw init {{preset_name}}`
```
Comment (andrik) [inline pages/common/jigsaw.md]:
```suggestion
`jigsaw init {{template_name}}`
```

File: pages/common/jigsaw.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# jigsaw
+
+> A Laravel-based static site builder for PHP.
+> More information: <https://jigsaw.tighten.co>.
+
+- Initialise a project:
+
+`jigsaw init`
+
+- Initialise a project using a starter template:
```
Comment (schneiderl) [inline pages/common/jigsaw.md]:
```suggestion
- Initialise a project using a starting template:
```

Comment (schneiderl) [reply pages/common/jigsaw.md]: Maybe this makes more sense?

Comment (mebeim) [reply pages/common/jigsaw.md]: Actually "starter", being an adjective, is more appropriate here.

Comment (schneiderl) [reply pages/common/jigsaw.md]: Makes sense 👍

**Pull request merged**